packages/access-api/src/durable-objects/space-verifier.js

```diff
@@ -0,0 +1,131 @@
+import { stringToDelegation } from '@web3-storage/access/encoding'
+
+/**
+ *
+ * @template {import('@ucanto/interface').Capabilities} [T=import('@ucanto/interface').Capabilities]
+ * @param {import('../bindings').DurableObjectNamespace} spaceVerifiers
+ * @param {string} space
+ * @param {import('@web3-storage/access/src/types').EncodedDelegation<T>} ucan
+ */
+export async function sendDelegationToSpaceVerifier(
+  spaceVerifiers,
+  space,
+  ucan
+) {
+  const durableObjectID = spaceVerifiers.idFromName(space)
+  const durableObject = spaceVerifiers.get(durableObjectID)
+  // hostname is totally ignored by the durable object but must be set so set it to example.com
+  const response = await durableObject.fetch('https://example.com/delegation', {
+    method: 'PUT',
+    body: ucan,
```

Comment:
Does body need to be a string or can we pass binary data ? If we could pass binary it might be a better option.

Reply:
ah good question, I suspect binary would work? noted and a good change for the next version of this I think